theme/less/_navigation.less

```diff
@@ -104,16 +110,14 @@
 #main-navigation li.has-children > a:after
 {
   content: url('/images/arrow_bullet.png');
-  padding-left: 11px;
-  vertical-align: sub;
+  padding: 0 0 0 11px;
+  vertical-align: middle;
 }
 
 #main-navigation ul ul li.has-children a:after
 {
-  position: absolute;
-  right: 10px;
-  top: 33%;
-  padding-right: 10px;
+  line-height: 22px;
+  padding: 0 13px 0 0;
```

Comment:
Suggestion:
```
  padding: 0 13px;
  height: 15px;
```

`line-height: 22px;` makes sub-items height bigger. What you want to do here is to make the height of the icon equal to the image height and align the item to the middle.

Let's make the paddings equal from both sides, it will prevent text comming to close to the icon and also in future if we decide to support RTL languages, might be a good idea to make it even.